lib/chef/resource/build_essential.rb

```diff
@@ -151,6 +152,20 @@ def xcode_cli_installed?
           # pkgutil returns an error if the package isn't found aka not installed
           cmd.error? ? false : true
         end
+
+        #
+        # Return to package label of the latest XCode Command Line Tools update, if available
+        #
+        # @return [String]
+        def xcode_cli_package
+          cmd = <<-EOH.gsub(/^ {14}/, "")
+          softwareupdate -l | grep "\*.*Command Line" | tail -n 1 | awk -F"*" '{print $2}' | sed -e 's/^ *//' | tr -d '\n' | sed 's/Label: //g'
+          EOH
+          cmd = Mixlib::ShellOut.new(cmd)
+          cmd.run_command
+          cmd.error!
```

Comment:
you should just use `shell_out!(cmd).stdout` here

Reply:
Got it!